### DISCUSSION TITLE
Disable mmuart0 interrupts to hart0 post FSBL

### DISCUSSION THREAD

**Discussion by Bharris100** [polarfire-soc/polarfire-soc-documentation#305] [How to] [answered]
I am using a Bare metal MSS payload running in DDR in my custom board application.  In my embedded software, Hart1 is using UART0 for command communication to external sources such as extraPuTTy and custom software.  Approximately 20% of the commands written to the embedded software respond with a "Unknown Command" response and one or two characters written from the UI.  The only place I can find the "unknown command" response is in the HSS (2022.8 with minor modifications in the ymodem loader file).  There is no such response in the MSS payload.  I have added the CONFIG_UART_SURRENDER=y in the .config for the HSS without success.  How do I disable interrupts to hart0 from UART0?

**Comment by griffini** [comment, marked as answer]
Make sure in your .config that `CONFIG_UART_SURRENDER=y` and `CONFIG_SERVICE_TINYCLI_REGISTER` is not set.

````
#CONFIG_SERVICE_TINYCLI_REGISTER is not set
CONFIG_UART_SURRENDER=y
````

If `CONFIG_SERVICE_TINYCLI_REGISTER=y`, `CONFIG_UART_SURRENDER` will implicitly be disabled.

**Reply by Bharris100** [reply]
Thank you Ivan, this seems to be working
